integration-tests/reactive-messaging-rabbitmq-vault/src/main/java/io/quarkus/it/rabbitmq/Person.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.it.rabbitmq;
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
+public class Person {
+    private String name;
```

Comment:
any reason to not use a public String and remove the getter and setter.

Reply:
Not really. Again I was just copying the `reactive-messaging-rabbitmq` test, which is itself basically a copy of `reactive-messaging-amqp`.

Reply:
Done